spec/dummy_driver.cr

```diff
@@ -163,6 +163,29 @@ class DummyDriver < DB::Driver
       read(String).to_f64
     end
 
+    def read(t : Time.class)
```

Comment:
These changes to the dummy_driver are not been used. For the sake of just-compiling the code could be just smaller.

If this code is used for some specs that is a different story for sure.